.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
@indexzero  can you please confirm, are you talking about Loggly free account? In this case, we  may need a work email address. 

I am suggesting to make mock test cases which will remove the service dependency.